assets/js/components/ClusterDetails/ClusterSettings.jsx

```diff
@@ -17,20 +18,26 @@ import { ConnectionSettings, ClusterInfoBox } from '@components/ClusterDetails';
 
 export const UNKNOWN_PROVIDER = 'unknown';
 
-export const ClusterSettings = () => {
+export const ClusterSettings = ({ newChecksSelectionView = false }) => {
   const { clusterID } = useParams();
 
   const cluster = useSelector(getCluster(clusterID));
 
   const tabsSettings = {
-    'Checks Selection': (
+    'Checks Selection': newChecksSelectionView ? (
+      <ChecksSelectionNew clusterId={clusterID} cluster={cluster} />
+    ) : (
       <ChecksSelection clusterId={clusterID} cluster={cluster} />
     ),
     'Connection Settings': (
       <ConnectionSettings clusterId={clusterID} cluster={cluster} />
     ),
   };
 
+  if (!cluster) {
```

Comment:
Not a biggie: would it make sense to move this guard right after the selection from the state?

Reply:
I prefer to leave it as it is to have consistency on order. `const` first and returned code afterwards.